src/containers/VisitPage.js

```diff
@@ -271,78 +277,74 @@ class VisitPage extends Component {
       right:  nextContent,
     };
 
-    const step = STEP_VALS[ stepIndex ];
-
-    const StepComponent = step.component;
+    let step          = STEP_VALS[ stepIndex ],
+        StepComponent = step.component;
 
     return (
-      <div className='forms-container flex-item flex-column'>
+      <div className={ `forms-container flex-item flex-column` }>
         {/* = PROMPTS & PROMPT TRIGGERS = */}
         {/* - Sometimes visible - */}
         {/* Triggered by `ReactRouterLeaveListener`,
          *`ResetAnytime`, or `ErrorListener` */}
         <FeedbackPrompt
           { ...this.state.promptData }
-          isBlocking={ this.state.isBlocking }
-          openFeedback={ this.openFeedback } />
+          isBlocking   = { this.state.isBlocking }
+          openFeedback = { this.openFeedback } />
         {/* Triggered by `FeedbackPrompt` & `FeedbackAnytime` */}
         <FeedbackForm
-          isOpen={ this.state.feedbackFormRequested }
-          close={ this.closeFeedback }
-          data={ this.state.client } />
+          isOpen = { this.state.feedbackFormRequested }
+          close  = { this.closeFeedback }
+          data   = { this.state.client } />
 
         {/* - Never visible - */}
         <ErrorListener
-          callback={ this.resetClientIfOk }
-          client={ this.state.client }
-          askForFeedback={ this.askForFeedback } />
+          callback       = { this.resetClientIfOk }
+          client         = { this.state.client }
+          askForFeedback = { this.askForFeedback } />
         {/* Browser nav - reload/back/unload. */}
         <BrowserLeaveListener isBlocking={ this.state.isBlocking } />
         {/* React nav buttons (Home/About) */}
         <ReactRouterLeaveListener
-          askForFeedback={ this.askForFeedback }
-          confirmer = { this.props.confirmer }
-          shouldRequestConfirmation={ this.shouldConfirmLeave }
-          isBlocking={ this.state.isBlocking } />
+          askForFeedback            = { this.askForFeedback }
+          confirmer                 = { this.props.confirmer }
+          shouldRequestConfirmation = { this.shouldConfirmLeave }
+          isBlocking                = { this.state.isBlocking } />
 
         {/* = LINKS? = */}
         {/* We should probably remove this. If we want to
          * do this we might do this a different way at this
          * point. Perhaps a user's page should be a route
          * in VisitPage? */}
-        { this.state.redirect ? (
+        { (this.state.redirect) ? (
```

Comment:
Why add the extra parens?

Reply:
Well, in more complex conditionals I find it easier to read if there are parens around the conditional. If I'm going to do it elsewhere, I thought I'd try being consistent and do it here too. Thoughts?

Reply:
I see. I kind of feel like it clutters things up to have extra parens. I think for conditionals that don't involve operations or multiple lines (for the conditional part), I would prefer leaving off the parens.